inc/saiswitch.h

```diff
@@ -1866,6 +1866,17 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_PACKET_EVENT_NOTIFY,
 
+    /**
+     * @brief DASH HA FSM event notification callback function passed to the adapter.
+     *
+     * Use sai_dash_ha_fsm_event_notification_fn as notification function.
+     *
+     * @type sai_pointer_t sai_dash_ha_fsm_event_notification_fn
+     * @flags CREATE_AND_SET
+     * @default NULL
+     */
+    SAI_SWITCH_ATTR_DASH_HA_FSM_EVENT_NOTIFY,
```

Comment:
since DASH is still SAI extension, it is better to move this into switch extension header we have in DASH.